src/components/useCase/UseCaseRecap/Component.astro

```diff
@@ -0,0 +1,31 @@
+---
+import { Wrapper } from '~/components/Wrapper';
+
+import s from './style.module.css';
+---
+
+<Wrapper>
+  <div class={s.root} data-component="UseCaseRecap">
+    {
+      Astro.slots.has('challenge') && (
+        <>
+          <div class={s.kicker}>The challenge</div>
+          <div class={s.text}>
+            <slot name="challenge" />
+          </div>
+        </>
+      )
+    }
+    {
+      Astro.slots.has('result') && (
+        <>
+          <div class={s.kicker}>The result</div>
+          <div class={s.text}>
+            <slot name="result" />
+          </div>
+        </>
+      )
+    }
+    <slot />
+  </div>
+</Wrapper>
```

Comment:
Prima:

![Screenshot 2024-10-30 at 15 23 31](https://github.com/user-attachments/assets/dac94d2d-e3e9-433e-ba00-850e512c0a29)


Dopo:


![Screenshot 2024-10-30 at 15 23 36](https://github.com/user-attachments/assets/d84a96dd-1bc3-44e8-9549-894a0275fea3)

Immagino che sia voluto, e ci può stare. Però:

1. <Numbers> non può stare dentro a <Wrapper>.. è brutto no? In /features/worldwide-cdn stava fuori dal wrapper
2. `var(--accent)` a questo punto diventa inutile, perchè si usa in soli due punti, e fa quasi strano introdurre un nuovo colore nella pagina per così poco utilizzo.

Reply:
Prima:

![Screenshot 2024-10-30 at 15 33 59](https://github.com/user-attachments/assets/dcd43de5-0ba4-4a77-8690-bea74802d30a)


Dopo:

![Screenshot 2024-10-30 at 15 33 53](https://github.com/user-attachments/assets/30f928c3-2549-49b1-b6e6-e1c301b8760d)

Reply:
Per quanto riguarda i numeri è stata una svista (:

Per l'immagine invece ho corretto la giustezza del testo, ma ho mantenuto una versione migliorata del gradiente invece del clip-path, perchè secondo me facilita la lettura.